specification/ai/Azure.AI.Projects/evaluations/models.tsp

```diff
@@ -68,7 +68,7 @@ model Evaluation {
 
   @doc("Status of the evaluation. It is set by service and is read-only.")
   @visibility(Lifecycle.Read)
-  status?: string;
+  status?: EvaluationStatus;
```

Comment:
Under what circumstances would there not be a status returned from the service? Presumably this should be:

```suggestion
  status: EvaluationStatus;
```

Reply:
Reverted my change. This was left as optional for a different use case where local runs submitted might not have a status. But that is outside the scope of this proposal